tests/pytorch/fused_attn/test_fused_attn.py

```diff
@@ -76,6 +76,12 @@ def _cudnn_version() -> Tuple[int, int, int]:
     return (major, minor, patch)
 
 
+def get_device_compute_capability() -> Tuple[int, int]:
```

Comment:
We have `from transformer_engine.pytorch.utils import get_device_compute_capability` at the top, so this function is redundant I think.

Reply:
idk why I pushed this file, I removed it, thanks